src/westpa/core/binning/mab.py

```diff
@@ -103,6 +110,12 @@ def map_mab(coords, mask, output, *args, **kwargs):
                     flipdifflist[n] = fliptemp[i][0]
                     flipmaxdiff = flipdiff
 
+    if splitting:
```

Comment:
Might be worth doing this in `log.info()` instead of `print` so it's not too much information for normal users. This way, these lines will only show if users run `--verbose` or `--debug` with `w_run`.

Make sure to add the following lines up top if you decided to!
```
import logging
log = logging.getLogger(__name__)
```